.github/workflows/ci.yml

```diff
@@ -149,6 +153,18 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e # v2.4.2
 
+      - name: Validate build settings
```

Comment:
This is only validation one thing, but we should consider adding more steps here overtime.